From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@SeanFeldman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

SeanFeldman commented:
I'd also flip between the columns in the table. First show the standard, then the premium.

tysonn commented:
@sethmanheim - Seth, could you review the update and suggestions in the comments from @SeanFeldman in the public repo?

sethmanheim commented:
@tysonn OK to merge, @SeanFeldman thanks for the contribution!

Commit:
Update functions-create-github-webhook-triggered-function.md

I get stuck because after forking that sample project the Issues tab was not shown.

amitbapat committed:
Added throttle limits for ECC key types

Commit:
add python module tutorial

Commit:
Merge branch 'master' into master

Commit:
minor

paulathurman committed:
Update log-analytics-network-performance-monitor.md

paulathurman committed:
Update log-analytics-network-performance-monitor.md

Commit:
minor updates

paulathurman committed:
Update log-analytics-network-performance-monitor.md

Commit:
add table and graph section

Commit:
add table and graph to storage-explorer.md

Commit:
add table and graph to storage-explorer.md

Commit:
update table, vertex pic

Commit:
solution tutorial

Commit:
update TOC

paulathurman committed:
edit pass: network-performance-monitor-articles

paulathurman committed:
edit pass: network-performance-monitor-articles

paulathurman committed:
fixing merge conflicts

paulathurman committed:
edit pass: network-performance-monitor-articles

paulathurman committed:
edit pass: network-performance-monitor-articles

paulathurman committed:
edit pass: network-performance-monitor-articles

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security

rwike77 committed:
updated toc

rwike77 committed:
removing draft https tutorial article

rwike77 committed:
fixed link

rwike77 committed:
added authentication and authorization

rwike77 committed:
added intro

cephalin committed:
revert to VS only steps

Commit:
add emulstor, update connect string, update pics

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security

rwike77 committed:
review updates

rwike77 committed:
moved article in TOC

Commit:
indent of pics

mikeurnun committed:
fixes #6047

fixes MicrosoftDocs/azure-docs#6047

Commit:
indent of pics

Commit:
indent of pics3

Commit:
resolve doc issues. update filter code for py3

Commit:
minors

Commit:
minor

Commit:
minor

Commit:
minor

genlin committed:
update heading

Commit:
cosmos DB troubleshoot

Commit:
cosdb troubleshoot score

Commit:
cosmosdb troubleshoot score2

Commit:
cosdb troubleshoot score3

Commit:
cosdb troubleshoot score4, update the keyword

Commit:
cosdb troubleshoot bold

Commit:
cosmosdb troubleshoot

Commit:
cosmosdb troubleshoot pic

Commit:
cosdb troubleshoot links

Commit:
cosdb troubleshoot links

Commit:
Added a section about Repair Manager

Introductory paragraph added for repair manager

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security

Commit:
Corrected link

Commit:
Fixed acrolinx issues

Commit:
azure -> Azure

rwike77 committed:
minor updates to overview

rwike77 committed:
updated metadata

Commit:
automation : update windows vm monitoring article

mikeurnun committed:
fixes #5830

fixes MicrosoftDocs/azure-docs#5830

Commit:
fixed links

rwike77 committed:
moved conceptual articles in to reliable services

itechedit committed:
Merge branch 'master' into consulting-services.md

paulathurman committed:
fixing merge conflicts

Commit:
cosdb troubleshoot links

Commit:
cosdb troubleshoot links

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into master304

Commit:
cosdb troubleshoot links

Commit:
Incorporated review comemnts

Added link to durability characteristics of cluster

Commit:
type issue

Commit:
type issue

genlin committed:
add refernece link

MikeDodaro committed:
Update paths to Nuget packages

MikeDodaro committed:
Update Nuget path in overview pages

kgremban committed:
Clarifying hostname error root cause

rwike77 committed:
added remote connect article

rwike77 committed:
updated article name

rwike77 committed:
moved images

Commit:
updates for the rebranding of OMS

Commit:
update debug doc

tfosmark committed:
ms.date change

Corrected to current year

itechedit committed:
Update consulting-services.md

tfosmark committed:
ms.date change

Corrected to current year

curtand committed:
[Azure AD groups] chnaged null to  in code and examples

CarlRabeler committed:
[sql db] get started connect and query

v-geberr committed:
move key concepts into concept page

Commit:
Merge branch 'master' into v-craic0322

nitinme committed:
reorganizing code output

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

rwike77 committed:
fixed links

v-geberr committed:
edits

nitinme committed:
another one

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

v-geberr committed:
toc

nitinme committed:
one more update

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

kgremban committed:
Added DNS details

Commit:
updates

nitinme committed:
some more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
last update

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

v-geberr committed:
edits

markgalioto committed:
fixing backup-azure-vms-introduction issue per customer feedback

JasonWHowell committed:
Update time-series-insights-manage-reference-data-csharp.md

nitinme committed:
This commit fixes #6065

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

rwike77 committed:
fixed conflict

JasonWHowell committed:
Update time-series-insights-manage-reference-data-csharp.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
another one

v-geberr committed:
edits

rwike77 committed:
acrolinx updates

douglaslMS committed:
Added new section with multiple screenshots.

douglaslMS committed:
Made one Acrolinx fix.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

PRMerger12 committed:
Merge pull request #36170 from nitinme/nitinme-databricks-updates

Fixing code output

markgalioto committed:
fixing the art in the file to align with changes to article

rmca14 committed:
Merge pull request #36047 from georgewallace/autoupdatetutorialupdate

automation : update windows vm monitoring article

kgremban committed:
Added image, updated --nopass

PRMerger14 committed:
Merge pull request #36184 from JasonWHowell/patch-3

Add nuget references to Fix MicrosoftDocs/azure-docs#5727

douglaslMS committed:
Fixed one image link and fixed Note syntax.

nitinme committed:
Another fix for github issue # 5324

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

markgalioto committed:
updating one last mention of temporary disk

CarlRabeler committed:
redirect

rmca14 committed:
Merge pull request #35705 from cephalin/quickstart

revert quickstart

rmca14 committed:
Merge pull request #36064 from rwike77/tocupdate

moved conceptual articles in to reliable services

ktoliver committed:
Merge pull request #36185 from nitinme/nitinme-adls-updates

Fixing Github issue #6065

CarlRabeler committed:
Update sql-database-connect-query.md

Commit:
Merge pull request #35388 from amitbapat/ecc-throttle-limits

Added ECC throttle limits

Commit:
Merge pull request #28725 from carolinacmoravia/patch-489

M16504: Possible typo: "and Model"

ktoliver committed:
Merge pull request #36186 from douglaslMS/adf-monitor-alerts

Added new section with multiple screenshots.

PRMerger10 committed:
Merge pull request #35883 from mike-urnun-msft/patch-1

fixes #6047

PRMerger12 committed:
Merge pull request #36050 from mike-urnun-msft/patch-2

fixes #5830

rwike77 committed:
fixed conflict

kgremban committed:
Update troubleshoot.md

v-geberr committed:
edits

rwike77 committed:
addressed review comments

Commit:
Sync

rmca14 committed:
Merge pull request #36166 from CarlRabeler/connect

[sql db] get started connect and query

ggailey777 committed:
Clarified @ericbrunner addition

ggailey777 committed:
Added a screenshot

Commit:
Fix links

kraigb committed:
Typo

ggailey777 committed:
Clarified table

ggailey777 committed:
Fixed image

amanarneja committed:
Fixes MicrosoftDocs/azure-docs#6187

PRMerger13 committed:
Merge pull request #36190 from amanarneja/patch-21

Fixes MicrosoftDocs/azure-docs#6187

Commit:
automation : hrw update

PRMerger19 committed:
Merge pull request #36189 from kraigb/kraigb-linkfix2

Fix links/bookmarks

rboucher committed:
Update monitoring-overview-unified-alerts.md

update for gartner

PRMerger12 committed:
Merge pull request #36192 from rboucher/patch-14

Update monitoring-overview-unified-alerts.md

PRMerger20 committed:
Merge pull request #36191 from georgewallace/hrwupdate

automation : hrw update

rboucher committed:
Update monitoring-overview-unified-alerts.md

Auto accept update I hope.  Updating text for Gartner update.

rboucher committed:
Update monitoring-overview-unified-alerts.md

PRMerger13 committed:
Merge pull request #36194 from rboucher/patch-14

Update monitoring-overview-unified-alerts.md

rboucher committed:
remove preview

PRMerger17 committed:
Merge pull request #36195 from rboucher/patch-15

remove preview

rboucher committed:
Update monitoring-overview-unified-alerts.md

rboucher committed:
Update monitoring-overview-unified-alerts.md

remove last preview

Commit:
Merge pull request #36196 from rboucher/patch-16

Update monitoring-overview-unified-alerts.md

PRMerger17 committed:
Merge pull request #36003 from novino/master

Added a section about Repair Manager

rayne-wiselman committed:
adding link to arch and support matrix

Commit:
Merge pull request #36198 from rayne-wiselman/rayne-march24

adding link to arch and support matrix

rayne-wiselman committed:
fixing outstanding PRs

PRMerger18 committed:
Merge pull request #36199 from rayne-wiselman/rayne-march24-2

fixing outstanding PRs

AjayKumar-N committed:
(ForumsTeam) Penetration testing broken link

In the referenced document https://docs.microsoft.com/en-us/azure/app-service/app-service-web-configuration-and-management-faq#how-do-i-perform-penetration-testing-for-my-app-service-app

The FAQ "How do I perform penetration testing for my App Service app?"
The answer - has a broken link and has to be fixed with this URL: https://portal.msrc.microsoft.com/en-us/engage/pentest

tdykstra committed:
reassign ownership

Commit:
Fixed bookmark issue

Commit:
Merge pull request #36203 from iRaindrop/vbruham_buildrptcleanup_B

Build report cleanup - bookmarks - batch B

Commit:
Merge pull request #36202 from tdykstra/patch-26

reassign ownership

Commit:
bookmark fixes and missing graphic

Commit:
bookmark fixes

Commit:
typo fixes

Commit:
link fix

Commit:
path fix

PRMerger15 committed:
Merge pull request #36206 from iRaindrop/vbruham_buildrptcleanup_D

Build report cleanup - bookmarks - batch D

v-geberr committed:
qs machine learned entities

PRMerger15 committed:
Merge pull request #36207 from v-geberr/0324-quickstart-intents-and-entities-rename

qs machine learned entities

Commit:
Bookmark fixes

Commit:
link corrections

Commit:
bookmark path fix

christiankuhtz committed:
Tweak

gsacavdm committed:
Add language to code blocks

gsacavdm committed:
Fix bad json

PRMerger14 committed:
Merge pull request #36210 from christiankuhtz/patch-106

Tweak

gsacavdm committed:
Fix indentation on code blocks

ponatara committed:
New change

GitHubber17 committed:
Merge pull request #34309 from rwike77/security

Application and service security articles

GitHubber17 committed:
Merge pull request #36138 from kgremban/mar23-5630

Clarifying hostname error root cause

PRMerger12 committed:
Merge pull request #36211 from gsacavdm/patch-2

Add language to code blocks

GitHubber17 committed:
Sentence case for headings

MikeDodaro committed:
Take out Preview text in all references to C# SDK

GitHubber17 committed:
Merge pull request #36183 from markgalioto/323b2018

Updating the Backup VMs intro article per GitHub Issue 5865

GitHubber17 committed:
Merge pull request #36167 from v-geberr/0323-key-concept

authoring and endpoint key concepts

GitHubber17 committed:
Merge pull request #36188 from ggailey777/5263

 Clarified @ericbrunner addition from public repo

GitHubber17 committed:
Merge pull request #36204 from iRaindrop/vbruham_buildrptcleanup_C

Build report cleanup - bookmarks - batch C

GitHubber17 committed:
Merge pull request #36212 from gsacavdm/patch-3

Fix bad json

mimig1 committed:
Cosmos DB: Incorporated usability study findings

ninarn committed:
Updating documentation on query editor to be accurate to the latest limitations

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vbruham_buildrptcleanup_E

PRMerger13 committed:
Merge pull request #36216 from ninarn/master

Updating documentation on query editor to be accurate to the latest l…

Commit:
Acrolinx fixes

Commit:
bookmark fix

Commit:
Merge pull request #36209 from iRaindrop/vbruham_buildrptcleanup_E

Build report cleanup - bookmarks - batch E

Commit:
Fixed bookmark issues

PRMerger17 committed:
Merge pull request #36219 from iRaindrop/vbruham_buildrptcleanup_F

Build report cleanup - bookmarks - batch F

ponatara committed:
new Change

ponatara committed:
New change-

Commit:
Merge pull request #36213 from ponatara/master

New change

Commit:
update as feedback

rayne-wiselman committed:
fixing outstanding public PR

Commit:
Merge pull request #36222 from rayne-wiselman/rayne-march26

fixing outstanding public PR

Commit:
update as feedback

Commit:
update as feedback

dominicbetts committed:
Address issue 5597

msvijayn committed:
Interface edits

msvijayn committed:
pic cropping

msvijayn committed:
Updating behavior

msvijayn committed:
Note not needed

Commit:
Merge pull request #36223 from dominicbetts/twin-issue

Address issue 5597 - device twin config

Commit:
Added bookmark.

PRMerger12 committed:
Merge pull request #36227 from jimdial/patch-112

Added bookmark.

PRMerger14 committed:
Merge pull request #36154 from craigcaseyMSFT/v-craic0322

updates for the rebranding of OMS

rwike77 committed:
updated next steps

Commit:
Updating docs with new default application debug mode

PRMerger12 committed:
Merge pull request #36229 from MikkelHegn/debugmode

Updating docs with new default application debug mode

mimig1 committed:
QA pass

Commit:
Fixed PEP credentials info

mimig1 committed:
readded db account creation include file

ktoliver committed:
Merge pull request #35988 from genlin/master304

azure-security-disk-encryption-faq.md

Commit:
Merge pull request #36230 from jeffgilb/326

Fixed PEP credentials info

Commit:
Merge pull request #36171 from curtand/null0323

[Azure AD groups] changed null to $null in code and examples

ktoliver committed:
Merge pull request #36148 from rwike77/nodetypes

Refactoring node types article

ktoliver committed:
Merge pull request #36200 from AjayKumar-N/patch-25

(ForumsTeam) Penetration testing broken link

itechedit committed:
Merge pull request #35495 from jingyanjingyan/storage-explorer

add table and graph section

mimig1 committed:
removed unused image

ktoliver committed:
Merge pull request #36224 from msvijayn/gartner-vijayn

Doc updated

itechedit committed:
Merge pull request #36132 from MikeDodaro/updateSDKlinks

Update paths to Nuget packages

itechedit committed:
Merge pull request #36215 from mimig1/03252018usability

Cosmos DB: Incorporated usability study findings

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ShawnJackson committed:
Merge pull request #34984 from paulathurman/network-performance-monitor-articles

edit pass: Network performance monitor articles

ShawnJackson committed:
Merge pull request #35338 from itechedit/consulting-services.md

edit pass: consulting-services.md

Commit:
remove changes in index file

v-alje committed:
Merge pull request #36237 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #35471 from shizn/master

Add python doc for IoT Edge

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
